bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx96-fr.html

```diff
@@ -0,0 +1,135 @@
+          </div>
+        </div>
+
+        <div class="mzp-c-picto">
+          <img class="mzp-c-picto-image" src="{{ static('img/firefox/whatsnew/whatsnew-96-eu/monitor.svg') }}" loading="lazy" alt="">
+          <h2 class="mzp-c-picto-heading">Firefox Monitor avertit de fuites de données connues</h2>
+          <div class="mzp-c-picto-body">
+            <p>Vérifiez si vous avez déjà été touché par une fuite de données et soyez averti si votre e-mail fait partie d’une fuite à l’avenir.</p>
+          </div>
+        </div>
+
+        <div class="mzp-c-picto">
+          <img class="mzp-c-picto-image" src="{{ static('img/firefox/whatsnew/whatsnew-96-eu/sync.svg') }}" loading="lazy" alt="">
+          <h2 class="mzp-c-picto-heading">La synchronisation aide à la navigation entre appareils</h2>
+          <div class="mzp-c-picto-body">
+            <p>Accédez à vos signets, mots de passe et onglets ouverts où que vous soyez. Avec la protection renforcée de la vie privée habituelle de Firefox.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="show-fxa-supported-signed-in">
+      <div class="mzp-l-content mzp-t-content-lg mzp-l-columns mzp-t-columns-three">
+        <div class="mzp-c-picto">
+          <img class="mzp-c-picto-image" src="{{ static('img/firefox/whatsnew/whatsnew-96-eu/fr/relay-extension.svg') }}" loading="lazy" alt="">
+          <h2 class="mzp-c-picto-heading">Utilisez l’extension Firefox</h2>
+          <div class="mzp-c-picto-body">
+            <p>Téléchargez l'extension Relay pour le navigateur Firefox et créez automatiquement des alias chaque fois qu'un site web vous demande votre adresse électronique. En un clic.</p>
+          </div>
+        </div>
+
+        <div class="mzp-c-picto">
+          <img class="mzp-c-picto-image" src="{{ static('img/firefox/whatsnew/whatsnew-96-eu/fr/relay-email-alias.svg') }}" loading="lazy" alt="">
+          <h2 class="mzp-c-picto-heading">Créez un nouvel alias</h2>
+          <div class="mzp-c-picto-body">
+            <p>Utilisez les alias d’e-mail lorsque vous ne souhaitez pas partager votre adresse e-mail privée. Par exemple, dans une boutique en ligne que vous ne connaissez pas proposant une offre intéressante. Ou une newsletter que vous souhaitez tester.</p>
+          </div>
+        </div>
+
+        <div class="mzp-c-picto">
+          <img class="mzp-c-picto-image" src="{{ static('img/firefox/whatsnew/whatsnew-96-eu/fr/relay-manage-address.svg') }}" loading="lazy" alt="">
+          <h2 class="mzp-c-picto-heading">Gérez vos alias</h2>
+          <div class="mzp-c-picto-body">
+            <p>Utilisez jusqu'à 5 adresses temporaires qui renvoient de manière sécurisée les messages à votre véritable adresse électronique jusqu'à ce que vous en ayez assez et que vous les désactiviez ou les supprimiez.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
```

Comment:
**nit:** similar to the `de` file, this should be a closing tag for the `section` element ✨